green-ecolution/green-ecolution-frontend
# Feature/watering plan overview

Close #273, 
refs #277

### How to test:
1. Navigate to the Overview page of »Einsätze«
2. Check if data is loaded
